library/std/src/io/mod.rs

```diff
@@ -746,7 +746,7 @@ pub trait Read {
     ///
     /// [`File`]s implement `Read`:
     ///
-    /// [`read`]: Read::read
+    /// [`read`]: Self::read
```

Comment:
Any reason you changed all these? I think it was more clear before.

Reply:
I'll change them back. They don't bother me at all but if they bother anyone I'll change them